indico/web/client/js/react/components/principals/util.js

```diff
@@ -144,6 +147,7 @@ export class PrincipalType {
       [PrincipalType.localGroup]: 0,
       [PrincipalType.multipassGroup]: 0,
       [PrincipalType.eventRole]: 1,
+      [PrincipalType.categoryRole]: 2,
```

Comment:
I wouldn't group it with users but event roles, so `1`. Another option would be keeping this as `2` and changing the user sort order to `3`.

Reply:
Going with the second option